manifests/init.pp

```diff
@@ -185,7 +185,7 @@
 
   $foreman_proxy_fqdn = $facts['networking']['fqdn']
   $foreman_url = $foreman_proxy::foreman_base_url
-  $reverse_proxy_real = $pulp or $reverse_proxy
+  $reverse_proxy_real = ($pulp or pulpcore_mirror) or $reverse_proxy
```

Comment:
Is this missing some 💲💲💲 before `pulpcore_mirror`?

Reply:
Yes, I should merge things in the evening, especially if they don't have tests.